docs/snippets/react/rsc-feature-flag.js.mdx

```diff
@@ -0,0 +1,9 @@
+```js
```

Comment:
Once again, the same formatting should be applied here and also for the associated variant.